app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use App\Models\User;
+use App\Models\Authentication;
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Foundation\Auth\RegistersUsers;
+use Illuminate\Auth\Events\Registered;
+use Illuminate\Support\Facades\DB;
+use Cookie;
+use App\Http\Requests\UserRequest;
+
+
+class UserController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    //urlを押した後に開く登録用の画面
+    public function index(Request $request)
+    {
+        $token = $request->token;  
+        $authentication = Authentication::where('token', $token)->first();
+
+        $datetime1 = now();      //現在時間 Carbon('now'); を省略した形
+        $datetime2 = $authentication->expired_at;  
+
+       if ($datetime1 < $datetime2 ) {
```

Comment:
```suggestion
       if ($datetime1 < $datetime2) {
```
条件式の中では演算子の前後のみにスペースを入れるようにお願いします。

Reply:
条件式内の不要なスペースを削除しました。